Add-README-files-3/8 (#5819)
TheAlgorithms/Python: digital_image_processing/README.md

# Digital Image Processing

> Digital Image Processing (DIP) is when you are processing and applying changes to an image with a computer.

Reviewer: ```suggestion
Digital Image Processing (DIP) is the process of applying changes to an image digitally
```
